src/en/foundations/colour/usage.md

```diff
@@ -12,31 +12,66 @@ eleventyNavigation:
   alt: This is an image of the foundation
 permalink: /en/foundations/colour/
 translationKey: "colour"
-tags: ['colourEN', 'usage']
+tags: ["colourEN", "usage"]
+type: color
+headers:
+  color-preview: ""
+  token-name: Token name
+  value: Hex
+  rgb: RGB
+  use-case: Use case
+useCases:
```

Comment:
The use cases to display in the column of type `use-case` 🤔 We could name it to match better